### PR TITLE
london-8/Laleh Koupaei/Javascript-coursework- week-2

### PR DESCRIPTION
**Pull request by lalehkoupa**
I have finished the exercise and Mandatory sections and will finish Extra later

**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

# Notes

- What did you find easy?

- What did you find hard?

- What do you still not understand?

- Any other notes?

### CODE REVIEW

File: exercises/C-comparison-operators/exercise.js [hunk 1]
```diff
@@ -7,14 +8,14 @@
 
 let studentCount = 16;
 let mentorCount = 9;
-let moreStudentsThanMentors; // finish this statement
+let moreStudentsThanMentors = true; // finish this statement
```
Comment (Simon2828) [inline exercises/C-comparison-operators/exercise.js:11]: Try to use comparison operators in this exercise eg. `studentCount > mentorCount`. You can assign this expression to the `moreStudentsThanMentors` variable:
`let moreStudentsThanMentors = studentCount > mentorCount`
The value of `moreStudentsThanMentors` is `true`

File: exercises/C-comparison-operators/exercise.js [hunk 2]
```diff
@@ -7,14 +8,14 @@
...
 
 let roomMaxCapacity = 25;
-let enoughSpaceInRoom; // finish this statement
+let enoughSpaceInRoom = true; // finish this statement
```
Comment (Simon2828) [inline exercises/C-comparison-operators/exercise.js:14]: Can you use a comparison operator here?

File: exercises/E-conditionals/exercise.js [hunk 3]
```diff
@@ -6,8 +7,17 @@
   If Daniel is a student, print out "Hi, I'm Daniel, I'm a student."
 */
 
+function checkRole(name, role) {
+  if (role == "mentor") {
```
Comment (Simon2828) [inline exercises/E-conditionals/exercise.js:11]:
```suggestion
  if (role === "mentor") {
```

**Review by Simon2828 — COMMENTED** [review]
Nice job Laleh! You've made lots of progress learning Javascript 😁

File: exercises/C-comparison-operators/exercise.js (hunk 1 above)
Comment (lalehkoupa) [reply exercises/C-comparison-operators/exercise.js:11]: Thanks @Simon2828 for reviewing my code.Yes definitely I should use comparison,I just did change in my code.

File: exercises/C-comparison-operators/exercise.js (hunk 2 above)
Comment (lalehkoupa) [reply exercises/C-comparison-operators/exercise.js:14]: Done

**Pull request closed without merging**